core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.kt

```diff
@@ -83,20 +84,24 @@ open class KiwixWebView @SuppressLint("SetJavaScriptEnabled") constructor(
     clearCache(true)
     setWebViewClient(webViewClient)
     webChromeClient = KiwixWebChromeClient(callback, nonVideoView, videoView, this).apply {
-      setOnToggledFullscreen { fullscreen ->
-        setWindowVisibility(
-          if (fullscreen) View.SYSTEM_UI_FLAG_LOW_PROFILE else View.SYSTEM_UI_FLAG_VISIBLE
-        )
-        callback.onFullscreenVideoToggled(fullscreen)
-      }
+      setOnToggledFullscreen(object : ToggledFullscreenCallback {
+        override fun toggledFullscreen(fullscreen: Boolean) {
+          setWindowVisibility(
+            if (fullscreen) View.SYSTEM_UI_FLAG_LOW_PROFILE else View.SYSTEM_UI_FLAG_VISIBLE
```

Comment:
Instead of using `SYSTEM_UI_FLAG_LOW_PROFILE`, we should hide window insets and system bars manually as it was deprecated and vice versa for `SYSTEM_UI_FLAG_VISIBLE`.

Reply:
hi @gouri-panda ,

i have replaced this with ```WindowCompat.setDecorFitsSystemWindows()``` and ```WindowInsetsControllerCompat() ``` methods.